webapi/Models/Response/BotResponsePrompt.cs

```diff
@@ -46,19 +47,13 @@ public class BotResponsePrompt
     public string ChatHistory { get; set; } = string.Empty;
 
     /// <summary>
-    /// Preamble to the LLM's response.
+    /// The collection of context messages associated with this chat completions request.
+    /// See https://learn.microsoft.com/en-us/dotnet/api/azure.ai.openai.chatcompletionsoptions.messages?view=azure-dotnet-preview#azure-ai-openai-chatcompletionsoptions-messages.
     /// </summary>
-    [JsonPropertyName("systemChatContinuation")]
-    public string SystemChatContinuation { get; set; } = string.Empty;
-
-    /// <summary>
-    /// Raw content of the rendered prompt.
-    /// </summary>
-    [JsonPropertyName("rawContent")]
-    public string RawContent { get; set; } = string.Empty;
+    [JsonPropertyName("metaPromptTemplate")]
+    public ChatHistory MetaPromptTemplate { get; set; } = new();
```

Comment:
nit:
The name of this type makes the code slightly confusing.
Suggestion:
Replace `using Microsoft.SemanticKernel.AI.ChatCompletion;` above to `using CompletionModelChatHistory = Microsoft.SemanticKernel.AI.ChatCompletion.ChatHistory;` 
Replace `public ChatHistory MetaPromptTemplate { get; set; } = new();` to `public CompletionModelChatHistory MetaPromptTemplate { get; set; } = new();`

Reply:
Totally agree, this was also bugging me in development. Renamed all cases of `Microsoft.SemanticKernel.AI.ChatCompletion.ChatHistory` to `ChatCompletionContextMessages` for clarity